.github/workflows/ubuntu_22.yml

```diff
@@ -581,10 +581,10 @@ jobs:
       device: 'igpu'
       test_type: ${{ matrix.TEST_TYPE }}
       runner: "[ 'self-hosted', 'igpu' ]"
+      runner-group: 'Intel-GPU'
       image: ubuntu:22.04
-      options: "--group-add 109 --group-add 44 --device /dev/dri:/dev/dri"
+      options: "--group-add 44 --group-add 993 --device /dev/dri/card1:/dev/dri/card1 --device /dev/dri/renderD128:/dev/dri/renderD128"
     # if: fromJSON(needs.smart_ci.outputs.affected_components).GPU
```

Comment:
We can also un-comment this line (it will still be executed in pre-commit for CI changes)